doc/start/get-involved.rst

```diff
@@ -12,11 +12,11 @@ These are exciting times in the Ceph community! Get involved!
 | **Blog**             | Check the Ceph Blog_ periodically to keep track | http://ceph.com/community/blog/               |
 |                      | of Ceph progress and important announcements.   |                                               |
 +----------------------+-------------------------------------------------+-----------------------------------------------+
-| **Planet Ceph**      | Check the blog aggregation on Planet Ceph for   | http://ceph.com/community/planet-ceph/        |
+| **Planet Ceph**      | Check the blog aggregation on Planet Ceph for   | http://ceph.com/category/planet/              |
```

Comment:
i am able to access https://ceph.com/category/planet/, so probably we can s/http/https/ ?